### PR TITLE
Simplifying Makefile

### PR DESCRIPTION
**Pull request by RenegadeTech**
Using inbuilt make expansion to reduce LOC in Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -54,94 +59,16 @@ graph:
...
-webdevops/liquibase:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/liquibase
+	docker run -t -i --rm -p 8000 \
+		-v "$$(pwd)/documentation/docs/:/opt/docs" \
+		-e "VIRTUAL_HOST=documentation.docker" \
+		-e "VIRTUAL_PORT=8000" \
+		webdevops/sphinx sphinx-autobuild \
+		--poll -H 0.0.0.0 /opt/docs html
+
+webdevops/%:
```
Comment (RenegadeTech) [inline Makefile]: This uses inbuilt GNU make pattern matching to reduce LOC and make maintainability easier.

File: Makefile [hunk 2]
```diff
@@ -54,94 +59,16 @@ graph:
...
-
-webdevops/varnish:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/varnish
-
-webdevops/certbot:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/certbot
-
-webdevops/liquibase:
-	python ./bin/console docker:build --threads=auto --whitelist=webdevops/liquibase
+	docker run -t -i --rm -p 8000 \
```
Comment (RenegadeTech) [inline Makefile]: Separated instruction over several lines to make easier to read and fit within 120 character line limit.

**Comment by mblaschke** [comment]
Nice PR, thank you 🎉

**Comment by RenegadeTech** [comment]
More than welcome ❤️

**Pull request merged**